common/supplemental/languageGroup.xml

```diff
@@ -34,6 +34,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<languageGroup parent="cdd">ari cad kii paw wic</languageGroup>
 		<languageGroup parent="cel">br cy ga gd gv kw mga nrc obt owl sga wlm xbm xcb xce xga xpi</languageGroup>
 		<languageGroup parent="cmc">ace cje cjm hro huq jra rad rog</languageGroup>
+		<languageGroup parent="cpe">tpi</languageGroup>
```

Comment:
just remembered that this file is _generated_ but it can be checked when that BRS step is reached.